src/main/java/stsjorbsmod/powers/OldBookPower.java

```diff
@@ -0,0 +1,65 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.actions.ExhumeCardsAction;
+import stsjorbsmod.util.CardMetaUtils;
+
+public class OldBookPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(OldBookPower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    private AbstractCard card;
+
+    public OldBookPower(final AbstractCreature owner, AbstractCard card) {
+        super(STATIC);
+
+        this.isTurnBased = false;
```

Comment:
I like using a power to make it more obvious that the effect is in play, but I think it should probably not be dismissable by Amnesia, so it should probably use a `SPECIAL` type to avoid that.

Reply:
resolving conversation as this has been addressed.